git_cc/sync.py

```diff
@@ -78,6 +79,19 @@ def do_sync(self):
                                           self.dst_root):
                     copied_file_count += 1
         return copied_file_count
+    
+    def do_sync_empty_folders(self):
```

Comment:
If you're going to do this I wouldn't have a separate function, I would pass an argument to `do_sync` to avoid the duplication here.